src/main/java/org/folio/consortia/service/FolioTenantService.java

```diff
@@ -81,6 +93,47 @@ protected boolean tenantExists() {
     );
   }
 
+  private void updateLocalTenantShadowSystemUsers() {
+    if (!consortiaConfigurationService.isCentralTenantConfigurationExists()) {
+      log.info("The first time module is being installed");
```

Comment:
Skipping update shadow system user permissions, because the first time module is being installed
or something like that to mare message more informative

Reply:
Updated